src/eyeTracker.m

```diff
@@ -218,52 +166,54 @@
                     return
                 end
 
-                % record a few samples before we actually start displaying
-                % otherwise you may lose a few msec of data
+                % Record a few samples before we actually start displaying otherwise you may lose
+                %  a few msec of data.
                 WaitSecs(0.1);
 
-                % HERE START THE STIMULATION OF THE BLOCK
-                % to mark the beginning of the trial
+                % Mark the beginning of the trial, here start the stimulation of the experiment.
                 Eyelink('Message', 'SYNCTIME');
 
             case 'StopRecordings'
 
-                % STEP 8
-                % finish up: stop recording eye-movements,
-                % EyeLink Stop recording the block
+                %% Stop recording of eye-movements
+
+                % EyeLink Stop recording the block.
                 Eyelink('Message', 'BLANK_SCREEN');
```

Comment:
Maybe the message should be clearer that "blank_screen": maybe "stop_recording" ? :wink: